frame/collective/src/lib.rs

```diff
@@ -65,7 +65,7 @@ pub trait Trait<I: Instance=DefaultInstance>: frame_system::Trait {
 	type Origin: From<RawOrigin<Self::AccountId, I>>;
 
 	/// The outer call dispatch type.
-	type Proposal: Parameter + Dispatchable<Origin=<Self as Trait<I>>::Origin> + From<Call<Self, I>>;
+	type Proposal: Parameter + Dispatchable<Origin=<Self as Trait<I>>::Origin> + From<frame_system::Call<Self>>;
```

Comment:
can you elaborate?

Reply:
The original statement allowed us to pass extrinsics from the module as a `Proposal`. In the benchmark, I opted to use `System::remark` instead as my generic proposal, so I placed the requirement that I can convert from `frame_system::Call` to proposal. This trait should always be satisfied for all pallets afaik